tests/results/AbsolutePositionTest.py

```diff
@@ -73,3 +73,18 @@ def test_instantiation(self):
         self.assertEqual(uut.position, 5)
         self.assertEqual(uut.line, None)
         self.assertEqual(uut.column, None)
+
+    def test_ordering(self):
+        text1 = ('A Lannister always pays his debts',)
+        text2 = ('Hello from the other side',)
+        uut1 = AbsolutePosition(text1, 4)
+        uut2 = AbsolutePosition(text1, 8)
+        uut3 = AbsolutePosition(text2, 4)
+        uut4 = AbsolutePosition(text2, 4)
+
+        self.assertTrue(uut1 <= uut2)
```

Comment:
is this needed: `uut1 <= uut2` ? when you have `uut1 < uut2` and  `uut1 == uut2` (which is assertFalse)
also if you have `uut1 < uut2` its the same thing for `uut2 > uut1`